R/rename_with_counts.R

```diff
@@ -1,16 +1,25 @@
 #' Concatenates the count to the end specific variable value
```

Comment:
This could use some love. What's the simple, ~5-word explanation of what this function does?